Source/ImPlotWidgets/Public/HistogramWidget.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+#pragma once
+
+#include <Runtime/Core/Public/CoreMinimal.h>
+#include "ImPlotWidgets/Public/ImPlotWidget.h"
+#include <ImGui/Public/Widgets/ImGuiCustomWidget.h>
+#include <Runtime/SlateCore/Public/Widgets/SCompoundWidget.h>
+
+class IMPLOTWIDGETS_API SHistogramWidget : public SImPlotWidget
+{
```

Comment:
I'd be tempted to put a `using Super = SImPlotWidget;` statement (privately) inside the class to make the usage of the calls to the parents more Unreal-like.